CMakeLists.txt

```diff
@@ -31,6 +31,9 @@
 cmake_minimum_required(VERSION 3.5)
 project(pmemkv)
 
+# set version of pmemkv
+set(VERSION 0.1)
```

Comment:
We need to do something about versioning. It's broken in all of our cmake-based repos.